__tests__/performance.js

```diff
@@ -0,0 +1,54 @@
+const moment = require('moment');
+const { getSeconds, getHours } = require('date-fns');
+const dayjs = require('dayjs');
+const iterations = 1000000;
+
+const performanceTest = testFunction => {
+  const start = new Date();
+  for (let i = 0; i < iterations; i++) {
+    testFunction();
+  }
+  const time = new Date() - start;
+  console.info(`Execution time: ${time}ms`);
```

Comment:
You can use `pretty-ms` module for this.

Reply:
Steve is right, Date() will not be accurate in terms of checking the performance.
Try to use native `performance.now()` or the module steve suggested.
Another options can be console.time() and console.endTime()
https://developer.mozilla.org/en-US/docs/Web/API/Performance/now

Reply:
nodejs does not support `performance.now()` but i will try with `console.time()`

Reply:
I changed to `console.time()`